init.lua

```diff
@@ -1,12 +1,13 @@
 -- Import section
 local server = require("server")
 local hud = require("hud")
-
+local event = require("event")
```

Comment:
Is this even used?

Reply:
Artifact from trying to run the loop in the background.
It's not used.